src/theme/basic/index.scss

```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
         overflow: visible;
     }
 
+    .q-grid-icon-button[disabled][disabled] {
+        color: $icon-button-disabled;
+    }
+
+    .q-grid-pager {
+        font-family: Roboto, "Helvetica Neue", sans-serif;
+        padding: 11px 0;
+        height: 24px;
+        font-size: 12px;
+        color: $pager-primary;
+
+        .pager-select {
```

Comment:
all classes should start with q-grid - q-grid-pager-select in this case

Reply:
done in my last commit